packages/editor/src/browser/editor-manager.ts

```diff
@@ -138,7 +138,7 @@ export class EditorManager extends NavigatableWidgetOpenHandler<EditorWidget> {
 
     async open(uri: URI, options?: EditorOpenerOptions): Promise<EditorWidget> {
         const editor = await super.open(uri, options);
-        this.revealSelection(editor, options, uri);
+        setTimeout(() => this.revealSelection(editor, options, uri));
```

Comment:
Can you explain why this is necessary? If `milliseconds` is omitted, the timeout is `0`.

Reply:
Before this change, when selecting line in Refactor Preview pane, first time the file is opened in new tab, relevant line is not selected. That seemed to me like the less intrusive fix for this issue.

![7989](https://user-images.githubusercontent.com/71069170/99269336-eb868380-282e-11eb-9069-ce61b8b191ed.gif)

Reply:
You mean `setTimeout` guruantees that an editor in the proper condition to reveal selection. Please revert.

Reply:
Reverted